kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -748,10 +748,23 @@ These are just used by the other operators for shuffling local execution state a
 ```k
     syntax InternalOp ::= "#newAccount" Int
                         | "#newExistingAccount" Int
-                        | "#newFreshAccount" Int
  // --------------------------------------------
     rule <k> #newAccount ACCT => #newExistingAccount ACCT ... </k> <account> <acctID> ACCT </acctID> ... </account>
-    rule <k> #newAccount ACCT => #newFreshAccount ACCT    ... </k> [owise]
+    rule <k> #newAccount ACCT => .    ... </k>
+             <accounts>
+               ( .Bag
+                  =>
+                 <account>
+                    <acctID> ACCT </acctID>
+                    <balance>     0                  </balance>
+                    <code>        .Bytes:AccountCode </code>
+                    <storage>     .Map               </storage>
+                    <origStorage> .Map               </origStorage>
+                    <nonce>       0                  </nonce>
+                 </account>
+               )
+               ...
+             </accounts> [owise, preserves-definedness]
```

Comment:
By moving this account creation here, the `[owise]` guarantees that `ACCT` is not in the `<accounts>` map, and the rule preserves definedness. Previosly, with the account creation isolated inside `#newFreshAccount`, which could be in principle called from anywhere, this would not hold.